llvm/lib/IR/Verifier.cpp

```diff
@@ -3765,9 +3765,28 @@ void Verifier::visitCallBase(CallBase &Call) {
           "Return type cannot be x86_amx for indirect call!");
   }
 
-  if (Function *F = Call.getCalledFunction())
+  if (Function *F = Call.getCalledFunction()) {
     if (Intrinsic::ID ID = (Intrinsic::ID)F->getIntrinsicID())
       visitIntrinsicCall(ID, Call);
+    // TODO: Move this to target dependent IR verifier once we have it.
+    auto IsAMDGPUEntryFunctionCC = [](CallingConv::ID CC) -> bool {
```

Comment:
Reinventing utility function in backend, can move to calling convention header. And rename to something more generic, like isCallableCC or something